$NetBSD: patch-ab,v 1.3 2011/09/04 21:20:06 dholland Exp $

these patches correct invalid C code that declares variables after statements.

--- psycho_4.c.orig	2003-03-02 11:50:48.000000000 +1100
+++ psycho_4.c
@@ -361,6 +361,7 @@ void psycho_4_init (double sfreq, option
   }  
  
 
+  {
   /* Work out the partitions
      Starting from line 0, all lines within 0.33 of the starting
      bark are added to the same partition. When a line is greater
@@ -381,6 +382,7 @@ void psycho_4_init (double sfreq, option
     /* keep a count of how many frequency lines are in each partition */
     numlines[partition_count]++;
   }
+  }
 
   /* For each partition within the frequency space, 
      calculate the average bark value - cbval [central bark value] */
